server/plugin/subscriptions.go

```diff
@@ -284,66 +277,10 @@ func (p *Plugin) StoreSubscriptions(s *Subscriptions) error {
 	return nil
 }
 
-func (p *Plugin) GetExcludedNotificationRepos() ([]string, error) {
-	var subscriptions []string
-	value, appErr := p.API.KVGet(SubscribedRepoNotificationOff)
-	if appErr != nil {
-		return nil, errors.Wrap(appErr, "could not get subscriptions from KVStore")
-	}
-	if value == nil {
-		return []string{}, nil
-	}
-	err := json.NewDecoder(bytes.NewReader(value)).Decode(&subscriptions)
-	if err != nil {
-		return nil, errors.Wrap(err, "could not properly decode subscriptions key")
-	}
-	return subscriptions, nil
-}
-
-func (p *Plugin) StoreExcludedNotificationRepo(s string) error {
-	var repoNames, err = p.GetExcludedNotificationRepos()
-	if err != nil {
-		return errors.Wrap(err, "error while getting previous value of key")
-	}
-	isDer, _ := ItemExists(repoNames, s)
-	if len(repoNames) > 0 && !isDer {
-		repoNames = append(repoNames, s)
-	} else if len(repoNames) == 0 {
-		repoNames = append(repoNames, s)
-	}
-	b, err := json.Marshal(repoNames)
-	if err != nil {
-		return errors.Wrap(err, "error while converting subscriptions map to json")
-	}
-
-	if appErr := p.API.KVSet(SubscribedRepoNotificationOff, b); appErr != nil {
-		return errors.Wrap(appErr, "could not store subscriptions in KV store")
-	}
-
-	return nil
+func (p *Plugin) GetExcludedNotificationRepos(s Subscription) []string {
```

Comment:
Why is this a method of the plugin?

Reply:
Same pattern is being followed for all the declarations in same file. should I remove it?

Reply:
The other method depend on `Plugin` I think. Are there ones that don't?

Reply:
I had a close look. You are right; they are calling some member method. Fixed.